models/Document/Editable/Video.php

```diff
@@ -1045,6 +1028,30 @@ private function getHtml5Code($urls = [], $thumbnail = null)
         return $code;
     }
 
+    private static function getDurationString(float $duration): string
```

Comment:
Hi @blankse is there a reason why this function is static?

Reply:
@robertSt7 No, the method doesn't need the object. We can move it in a helper so we can reuse it. Or do you prefer hold it private without static?

Reply:
```suggestion
    private function getDurationString(float $duration): string
```

@blankse You need an instance to call this function, because it is private. I think we should remove the static. What is your opinion about that?

Reply:
@robertSt7 I changed it https://github.com/pimcore/pimcore/pull/15621/commits/7fbfc2a22af0102ca51a0cad64988828871707f3